docs/README.md

```diff
@@ -1,106 +1,131 @@
-# About Coder
+# What is Coder
 
-Coder is an open-source platform for creating and managing developer workspaces
-on your preferred clouds and servers.
+Coder is a self-hosted, open source, cloud development environment that works
+with any cloud, IDE, OS, Git provider, and IDP.
 
-<p align="center">
-  <img src="./images/hero-image.png">
-</p>
+![Screenshots of Coder workspaces and connections](./images/hero-image.png)_Screenshots of Coder workspaces and connections_
 
-By building on top of common development interfaces (SSH) and infrastructure tools (Terraform), Coder aims to make the process of **provisioning** and **accessing** remote workspaces approachable for organizations of various sizes and stages of cloud-native maturity.
+Coder is built on common development interfaces and infrastructure tools to
+make the process of provisioning and accessing remote workspaces approachable
+for organizations of various sizes and stages of cloud-native maturity.
 
-<blockquote class="warning">
-  <p>
-  If you are a Coder v1 customer, view <a href="https://coder.com/docs/coder">the docs</a> or <a href="https://coder.com/docs/coder/latest/guides/v2-faq">the sunset plans</a>.
-  </p>
-</blockquote>
+> **Using Coder v1?**
+>
+> If you're a Coder v1 customer, view [the v1 documentation](https://coder.com/docs/v1)
+or [the v2 migration guide and FAQ](https://coder.com/docs/v1/guides/v2-faq).
 
-## How it works
+## IDE support
 
-Coder workspaces are represented with Terraform, but no Terraform knowledge is
-required to get started. We have a [database](https://registry.coder.com/templates) of pre-made templates built into the
-product.
+![IDE icons](./images/ide-icons.svg)
 
-<p align="center">
-  <img src="./images/providers-compute.png">
-</p>
+You can use:
 
-Coder workspaces don't stop at compute. You can add storage buckets, secrets, sidecars
-and whatever else Terraform lets you dream up.
+- Any Web IDE, such as
+  - The built-in [VS Code code-server](https://github.com/coder/code-server)
```

Comment:
```suggestion
  - [code-server](https://github.com/coder/code-server)
```
Let's not link code-server with VSCode. VS Code has their own similar offering they call VSCode web.